descriptors/ModuleDescriptor-template.json

```diff
@@ -674,9 +674,29 @@
           "delay": "5"
         }
       ]
+    },
+    {
+      "id": "print-events-storage",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/print-events-storage/print-events",
```

Comment:
considering this is not a REST endpoint, I would name it differently, with a verb to accentuate its RPC nature. Something like `POST /print-events-storage/print-events/create-batch` (similar to `/loan-storage/loans/anonymize/{userId}`)